potofolio.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div id="logoplaceholder"><a href="#"></a></div>
```

Comment:
[OPTIONAL] I think it will be good if you add content to your logo, maybe an image or text or your name. 😉

Reply:
thank you